src/client/pages/OfferNew/quote-ids.ts

```diff
@@ -0,0 +1,44 @@
+import queryString from 'querystring'
+import { History } from 'history'
+import { useEffect, useState } from 'react'
+import { StorageState } from 'src/client/utils/StorageContainer'
+
+export const useQuoteIds = (storage: StorageState, history: History) => {
+  const [quoteIds, setQuoteIds] = useState<ReadonlyArray<string> | null>(null)
```

Comment:
Do we need to type this as `ReadonlyArray`? 🤔  I can't see why anyone would have the idea to mutate this array. But hey, maybe better safe than sorry? Or what was the reasoning here?

Reply:
I'm working on the principle of least privilege for the return type, so that anyone accidentally attempting to mutate the array get type errors 🙂